### ISSUE TITLE
README: missing dependencies section

### ISSUE THREAD

**Issue by luzpaz** [StefanHamminga/kde-fcstd-thumbnailer#1]
Great little find here. WIsh I knew about this before. Does it still work?

**Comment by mfraser** [comment]
Yes, it still works. Sorry can't help with missing dependencies section.

**Comment by StefanHamminga** [comment]
It builds just fine as long as you install `cmake-extra-modules`, but it doesn't seem to actually work with two files I plucked from KiCAD's 3D model library. This plugin opens a FreeCAD file as zip archive and extracts a PNG thumbnail which is used by Dolphin. The files I got are from 3 and 4 years ago and no longer have an embedded thumb (right-click -> open with `ark` to have a look).

**Comment by luzpaz** [comment]
Maybe we can ask for something like this to be included directly in to [dolphin](https://phabricator.kde.org/project/view/194/)?

**Comment by StefanHamminga** [comment]
Thumbnailers are stand-alone and not part of the Dolphin source afaik. But a universal 'extract thumb from zip-based file format' thumbnailer shouldn't be too hard. Do you have suggestions on which other file formats would work?

**Comment by StefanHamminga** [comment]
Coincidentally I opened a 3MF file (zip based) and it has an embedded PNG thumb, so that would be one.

**Comment by luzpaz** [comment]
Let continue the discussion on the forum thread. That way more of the FC community can be involved.

**Comment by luzpaz** [comment]
Per @wwmayer in https://forum.freecadweb.org/viewtopic.php?p=591499#p591499
> libkf5kio-dev contains the needed header thumbcreator.h

**Comment by luzpaz** [comment]
JFYI, we're encountering a missing dependency in the forum thread: https://forum.freecad.org/viewtopic.php?p=689687#p689687

Can any devs weigh-in ?

**Comment by yochule** [comment]
hi people, first of all, thanks for reading.
i'm trying to get the thumbnail working in my notebook. i'm running gnu-linux ubuntu 22.04LTS and installed kde-dolphin 21.12. 
so, i'm following [this thread](https://forum.freecad.org/viewtopic.php?t=68087). checking the compilation error messages i installed :

* cmake
* libkf5kdelibs4support5-bin
* build-essential
* extra-cmake-modules
* qtbase5-dev qt5-qmake
* libkf5kio-dev
* libkf5archive-dev

looks like i managed to install all dependencies needed, and finally installed the project without errors
but still, no thumbnail showed
any idea why is not working? suggestions?
